robber/matchers/boolean.py

```diff
@@ -11,7 +11,9 @@ def matches(self):
         return self.actual == True
 
     def failure_message(self):
-        return 'Expected %s to be True' % self.actual
+        negated_message = ' not' if self.is_negated else ''
+        return 'Expected %s%s to be True' % (self.actual, negated_message)
```

Comment:
If we move the space into the %s, when we don't have not, it will be something like this:
```
Expected False  to be true
```
When we just want this:
```
Expected False to be true
```